Makefile

```diff
@@ -154,11 +154,11 @@ multimod-verify: | $(MULTIMOD)
 	$(MULTIMOD) verify
 
 COREPATH ?= "../opentelemetry-collector-contrib"
-OPTIONAL_CORE_PATH ?="../opentelemetry-collector"
+SKIP_GO_MOD_TIDY ?= "true"
 .PHONY: multimod-sync-core
 multimod-sync-core: multimod-verify
-	@[ ! -d $COREPATH ] || ( echo ">> Path to core repository must be set in COREPATH and must exist"; exit 1 )
-	$(MULTIMOD) sync -a -o ${COREPATH} -o ${OPTIONAL_CORE_PATH}
```

Comment:
Another thing why I delete the ```OPTIONAL_CORE_PATH``` is: after sync-core from the first repo, the dependencies from the upstream repo are updated. However, every time it runs a sync-core, it needs to be verified [if there are any changes before](https://github.com/open-telemetry/opentelemetry-go-build-tools/blob/fdc91fd5380079ee89ea9315029a5b2304851b45/multimod/internal/sync/sync.go#L46-L48). Therefore, between showing an error when running twice ```make multimod-sync-core``` and run sync-core once repo at a time, latter seems more feasible for me

Reply:
If that's the case, then separate them out into separate targets: `multimod-sync-core` and `multimod-sync-contrib`.

Reply:
Any reasons why we should separate the two? Since ```multimode-sync-core``` can be use as ```make sync-core COREPATH=../opentelemetry-collector```

Reply:
Nice suggestion. Added and thanks for the call